src/devtools/client/debugger/src/actions/pause/fetchScopes.js

```diff
@@ -7,7 +7,7 @@
 import { getSelectedFrame, getFrameScope } from "../../selectors";
 import { PROMISE } from "ui/setup/redux/middleware/promise";
 
-export function fetchScopes(cx) {
+export function fetchScopes() {
```

Comment:
:speech_balloon: I'm still shaky on and we do and don't need this passed through

Reply:
It's only used by the "context middleware", as a central checking point.  (I think the idea is that some actions may be irrelevant if the app has moved on to a new pause situation by the time a response returns.)

Reply:
ah yes, I had forgotten that some middleware was referencing it. I think we should have other methods for solving these problems.